prow/cluster/deck_service.yaml

```diff
@@ -15,10 +15,12 @@
 apiVersion: v1
 kind: Service
 metadata:
-  labels:
-    app: deck
   namespace: default
   name: deck
+  labels:
+    app: deck
+  annotations:
+    cloud.google.com/neg: '{"ingress": true}'
```

Comment:
This annotation is required because this Service is a ClusterIP.

Reply:
For anyone wondering why this annotation is set:

> In the Service manifest, you must use `type: NodePort` unless you're using [container native load balancing](https://cloud.google.com/kubernetes-engine/docs/concepts/container-native-load-balancing). If using container native load balancing, use the `type: ClusterIP`.

For context,

```yaml
cloud.google.com/neg: '{"ingress": true}'
```

is the annotation used to tell ingress-gce to enable container native load balancing.